src/modules/cloudshelf/cloudshelf.api.util.ts

```
@@ -144,7 +156,7 @@ export class CloudshelfApiUtils {
 
     static async upsertStore(apiUrl: string, retailer: RetailerEntity, logs?: LogsInterface): Promise<void> {
         const timestamp = new Date().getTime().toString();
-        const authedClient = await this.getCloudshelfAPIApolloClient(apiUrl, retailer.domain);
+        const authedClient = await this.getCloudshelfAPIApolloClient(apiUrl, retailer.domain, logs);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)